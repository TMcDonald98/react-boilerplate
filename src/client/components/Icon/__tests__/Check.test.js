import * as React from 'react';
import { Check } from '../Check';

describe('components/Icon/Check', () => {
  describe('render', () => {
    it('should render without error', () => {
      const result = shallow(<Check />);

      expect(result).toMatchSnapshot('Check');
    });
  });
});
