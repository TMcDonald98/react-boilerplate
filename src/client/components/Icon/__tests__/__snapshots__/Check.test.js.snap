// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Icon/Check render should render without error: Check 1`] = `
<svg
  className="check-icon"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="#25A865"
    transform="translate(1 1)"
  >
    <circle
      cx="11"
      cy="11"
      r="11"
    />
    <path
      d="M4.349 11.408l4.066 5.269L17.51 6.069"
      strokeLinecap="round"
    />
  </g>
</svg>
`;
